lib/hutch/cli.rb

```diff
@@ -76,6 +76,7 @@ def load_rails_app(path)
           logger.info "found rails project (#{path}), booting app in #{ENV['RACK_ENV']} environment"
           require rails_path
           ::Rails.application.eager_load!
+          Hutch::Config.initialize(Hutch::Config.env_based_config)
```

Comment:
The code in line 14 is calling the initialize of Hutch config, that already loads the `env_based_config`. 
You mean that is not loading any config as Figaro/Dotenv were not loaded yet?

Apart of that, they recomend to use a separated config file. @h0lyalg0rithm

Reply:
@vikalejo they dont load the env config